parquet-protobuf/src/test/resources/TestProto3.proto

```diff
@@ -1,4 +1,3 @@
-syntax = "proto3";
```

Comment:
I changed this so that it is below the license information and to keep it in sync with `TestProto.proto`